app/main/views/feedback.py

```diff
@@ -0,0 +1,32 @@
+import requests
+
+from werkzeug.exceptions import ServiceUnavailable
+from werkzeug.datastructures import MultiDict
+from werkzeug.urls import url_parse
+
+from flask import current_app, request, redirect, flash, Markup
+
+from .. import main
+
+
+@main.route('/feedback', methods=["POST"])
+def send_feedback():
+    feedback_config = current_app.config['DM_FEEDBACK_FORM']
+    form_data = MultiDict()
+    for field, google_form_field in feedback_config['fields'].items():
+        form_data.setlist(google_form_field, request.form.getlist(field))
+
+    result = requests.post(feedback_config['uri'], list(form_data.items(multi=True)))
+    if result.status_code != 200:
+        raise ServiceUnavailable('Google forms submission problem (status %d)'.format(result.status_code))
+    came_from = url_parse(request.form['uri'])
+    # strip netloc and scheme as we should ignore attempts to make us redirect elsewhere
+    replaced = came_from._replace(scheme='', netloc='')
+
+    flash(Markup(
+        """Thank you for your message. If you have more detailed feedback, please email 
+        <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> or
+        <a href="https://airtable.com/shrkFM8L6Wfenzn5Q">take part in our research</a>.
+        """))
+
```

Comment:
![screen shot 2017-08-29 at 15 50 28](https://user-images.githubusercontent.com/4599889/29827585-43a48464-8cd2-11e7-9cec-2f0565478c83.png)
@karlchillmaid - this is how it will look

Reply:
Also @samuelhwilliams 's comment 
"How will we know if the airtable request expires (other than getting support requests about it)? We shouldn't use that link if it's liable to die on us with no warning."

Reply:
If it's just that we're changing from airtable to something else, that's probably fine - we just need to make sure we remember to update this link. :)